From 0b48d5eef23e4d072672e398cb2d5b31477a23d7 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Thu, 6 Sep 2018 15:21:44 +0300
Subject: [PATCH 292/706] arm64: dts: ls1028a: Add link properties to enetc
 port nodes for SIMU

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit 3801774ff122fe4a9f60b6d4eadba98a9166d38d)
(cherry picked from commit 6de793a8de8c431f804eb5b1b74298982738155b)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 .../boot/dts/freescale/fsl-ls1028a-simu.dts   | 32 +++++++++++++++++++
 1 file changed, 32 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-simu.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-simu.dts
index ae4b879afe02..db3a569eed9c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-simu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-simu.dts
@@ -59,3 +59,35 @@
 &dp0 {
 	status = "okay";
 };
+
+&enetc_port0 {
+	phy-connection-type = "internal";
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
+
+&enetc_port1 {
+	phy-connection-type = "internal";
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
+
+&enetc_port2 {
+	phy-connection-type = "internal";
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
+
+&enetc_port3 {
+	phy-connection-type = "internal";
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
-- 
2.17.1

